docs/development.rst

```diff
@@ -163,6 +163,32 @@ virtual env and run::
 
   pip install -r requirements.d/development.txt
 
+Building on Windows
++++++++++++++++++++
+
+Install scoop: https://scoop.sh/
+
+install MSYS2: ``scoop install msys2``
```

Comment:
I don't know scoop, is it only used for downloading and executing the msys2 installer?

Reply:
yes, it is a save way to  install linux ported programs on windows

Reply:
but if it is used only once (in our context), isn't it then easier and more direct to just download the msys2 installer as their homepage says?

Reply:
Installation of msys2 over scoop sets up env variables for msys2 and ming2, which is very handy.
Hence, openssl installation is now easy.

Reply:
ah, ok.